README.md

```diff
@@ -19,6 +19,8 @@ apt-get install -y make texlive-latex-base texlive-latex-recommended texlive-lat
 
 `make clean` will remove any generated files.
 
+Alternatively, or on OSes not derived from Debian, Docker may be installed and the PDF conversion achieved with: `sudo bash pdf_from_latex.sh`.
```

Comment:
maybe no sudo, but needed for now b/c docker container leaving generated files on host